User/User.Api/Controllers/AccountController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using User.Application.Dtos;
+using User.Application.Interfaces;
+
+namespace User.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountController : ControllerBase
+    {
+        private readonly IUserService _userService;
+        private const string Jwt_Security_Key = "averysecretkeygeneratedformicroserviceauth";
+        private const int Jwt_Token_Validity_Mins = 20;
+        public AccountController(IUserService userService)
+        {
+            _userService = userService;
+        }
+        [HttpPost]
+        public ActionResult<AuthenticationResponse> Authenticate(AuthenticationRequest request)
+        {
+            var response = GenerateJwtToken(request);
+            if (response == null) return Unauthorized();
+            return response;
+        }
+
+        private AuthenticationResponse GenerateJwtToken(AuthenticationRequest request)
+        {
```

Comment:
This is kind of a service which is being used 
and could have been implemented as an another service and not within Controller
for the seperation of concerns